### DISCUSSION TITLE
In documentation of celery 4, it is mentioned that class based tasks are not autoregistered anymore, but they are getting registered.

### DISCUSSION THREAD

**Discussion by anveshagarwal** [celery/celery#7104] [Q&A] [answered]
# Checklist

- [x] I have checked the [issues list](https://github.com/celery/celery/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22Category%3A+Documentation%22+)
  for similar or identical bug reports.
- [x] I have checked the [pull requests list](https://github.com/celery/celery/pulls?q=is%3Apr+label%3A%22Category%3A+Documentation%22)
  for existing proposed fixes.
- [x] I have checked the [commit log](https://github.com/celery/celery/commits/master)
  to find out if the bug was already fixed in the master branch.
- [ ] I have included all related issues and possible duplicate issues in this issue
      (If there are none, check this box anyway).

#### Related Issues

- None

#### Possible Duplicates

- None

# Description

I am using celery 4.3, and according to it's documentation, the class based tasks are not registered automatically (or autodiscovered) they have to be registered manually : https://docs.celeryproject.org/en/3.0/whatsnew-4.0.html#the-task-base-class-no-longer-automatically-register-tasks. But I see all class based tasks are getting registered in my case automatically. And now when I upgraded to celery 5.1.*, now the class based tasks are not getting registered. So in which version exactly this feature is removed?
# Suggestions

[a comment by an automated account is not shown]

**Comment by auvipy** [comment, marked as answer]
it was deprecated in v4 and removed in 5 as far i can recall
